wg-releases/wg-qa/README.md

```diff
@@ -21,16 +21,23 @@ Oversees stability, quality, and testing criteria within Electron.
 
 ## Current Objective and Key Results
 
-**Objective:**
+**Objective 1:**
 
-Electron stable releases should be of high quality and easy for apps to upgrade.
+Early detection of Electron issues
 
 **Key Results:**
-* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their GA product.
-* Measure the average or cumulative delta between an electron stable release, and the AFP participants shipping that same stable release in their canary product.
-* Reduce the size of outstanding AFP-raised issues at the time of stable.
-* Reduce the number of _would-be blockers_ discovered _after_ a stable ship.
-* Add 4 new members and 1 new organisation/company.
+
+* Report alpha/beta electron issues on latest 2 release versions and beta version (as a team)
+* Collaborate on and deliver a list of api tests (via fiddle?) that can be automated
```

Comment:
Does "collaborate" here mean a team effort or are we going to work with other teams/companies?
Do we have an idea of how long that list should be? Is it going to be a percentage of the API surface (e.g. 90%) or a raw number (maybe based on what we know are the most used ones)?

Reply:
Collaborate here means a team effort of the WG-QA.

To me, it makes more sense to base the list on the most used APIs.

Reply:
It also makes more sense to me 😊
Do we know how to get that information?

Reply:
* Audit current automated API tests and document results

Note: would be interesting to get some telemetry on most popular APIs